diff --git a/src/include/duckdb/main/database.hpp b/src/include/duckdb/main/database.hpp
index 7e43de6005..094ee08815 100644
--- a/src/include/duckdb/main/database.hpp
+++ b/src/include/duckdb/main/database.hpp
@@ -17,6 +17,9 @@
 #include "duckdb/main/extension_manager.hpp"

 namespace duckdb {
+
+extern bool preloaded_httpfs;
+
 class BufferManager;
 class DatabaseManager;
 class StorageManager;
